src/ape/managers/chain.py

```diff
@@ -441,6 +441,12 @@ def get(
         contract_type = self._get_contract_type_from_disk(address)
 
         if not contract_type:
+            # Contract could be a minimal proxy
+            if hasattr(self.provider.network.ecosystem, "proxy_info"):
+                proxy = self.provider.network.ecosystem.proxy_info(address)
```

Comment:
Can we name to `get_proxy_info()`?
`proxy_info` sounds more like a property name than a method